### PR TITLE
feat: Slider support editable

### PR DESCRIPTION
**Pull request by zombieJ**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

- Use a developer-oriented tone and narrative style.
- Describe the user's first-hand experience of the issue and its impact on developers, rather than your solution approach.
- Refer to: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Slider support `range.editable` to dynamic add / remove handles.        |
| 🇨🇳 Chinese |     Slider 支持 `range.editable` 以动态添加/删除节点。       |

### CODE REVIEW

**Comment by zombieJ** [comment]
@SocketSecurity ignore npm/sharp@0.33.4

**Comment by li-jia-nan** [comment]
删除节点之后 Tooltip 没有消失：

image

**Review by li-jia-nan — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -26,20 +26,12 @@
     "url": "https://opencollective.com/ant-design"
   },
   "license": "MIT",
-  "sideEffects": [
-    "*.css"
-  ],
+  "sideEffects": ["*.css"],
```
Comment (crazyair) [inline package.json:29]: 上次你的 pr 是换行，今天又不换行

Comment (zombieJ) [reply package.json:29]: 我艹？我真是好懵逼

Comment (li-jia-nan) [reply package.json:29]: 要跟我的 PR 冲突了

**Pull request merged**